adapters/dynamoDB_adapters/src/main/java/org/atomhopper/dynamodb/query/SqlBuilder.java

```diff
@@ -0,0 +1,82 @@
+package org.atomhopper.dynamodb.query;
+
+import com.amazonaws.services.dynamodbv2.document.spec.QuerySpec;
+import org.apache.commons.lang.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.List;
+import java.util.Map;
+
+public class SqlBuilder {
```

Comment:
It is _so_ weird that you can use SQL based queries for DynamoDB as a primary strategy. This isn't a request to chagne it, just a comment on... it's weird.